structure_tests/README.md

```diff
@@ -135,6 +135,13 @@ Structure tests can also be run outside of Cloud Container Build through a shell
 
 This script will retrieve the static structure_test binary from the published Docker image, volume mount it (along with all specified config files) into the container built from the image under test, and run the tests as normal. Supported arguments:
 
+You can fetch and run this script via:
+
+```shell
+curl https://raw.githubusercontent.com/GoogleCloudPlatform/runtimes-common/master/structure_tests/ext_run.sh > ext_run.sh
+bash ext_run.sh -i gcr.io/gcp-runtimes/check_if_tag_exists -c check_if_image_tag_exists/test_config.json
```

Comment:
nit: maybe don't include the line that actually runs the script, since we already have that just a few lines down? if you think it's more clear this way that's fine.